### DISCUSSION TITLE
Running Baby Buddy

### DISCUSSION THREAD

**Discussion by Kauetheadventurer246** [babybuddy/babybuddy#624] [Q&A]
Hello, I'm trying to figure out how to get Baby Buddy to work to test it out. However, the README doesn't really tell me how to actually get it to run. If you can give me a step by step guide it would be appreciated.

**Comment by cdubz** [comment]
The docs should have what you need -- https://docs.baby-buddy.net/setup/deployment/

**Reply by genericCog** [reply]
I'm just a lurker in the shadows on this repo, but the Original Poster's concern stems from the way the documentation is geared towards people who already have familiarity with building and hosting their own web apps. Imagine if you have never written a line of code, then see an awesome repo, but the Readme starts with a sudo command. The deep dive through Docker containerization is a barrier to entry unless a person has time to learn how to program from ground zero. The OP needs the documentation to be less technical and geared towards a family member with time constraints and core competencies in non-programming areas.

**Reply by cdubz** [reply]
Lurker comments welcome!

I think I make a general assumption of a relatively high level of technical experience/ability for anyone looking to _self host_... well, anything, but also something like Baby Buddy where one probably already has a good deal of additional stress going on 😄 

That said we did used to have the Heroku deploy method that was essentially a one-click. I took it away after Salesforce dropped their support for free tier usage but perhaps that was drastic. I'll look in to bringing that back... and also open to recommendations for other one-click solutions folks might want!
